projects/demo/src/modules/components/icon/index.ts

```diff
@@ -1,12 +1,28 @@
 import {Component} from '@angular/core';
 import {changeDetection} from '@demo/emulate/change-detection';
 import {TuiDemo} from '@demo/utils';
-import {TuiNotificationComponent} from '@taiga-ui/core';
+import {TUI_USED_ICONS} from '@taiga-ui/cdk';
+import {TuiIconComponent} from '@taiga-ui/core';
+import {tuiInputNumberOptionsProvider} from '@taiga-ui/legacy';
 
 @Component({
     standalone: true,
-    imports: [TuiDemo, TuiNotificationComponent],
+    imports: [TuiDemo, TuiIconComponent],
     templateUrl: './index.html',
     changeDetection,
+    providers: [
+        tuiInputNumberOptionsProvider({
+            min: 0,
+        }),
+    ],
 })
-export default class PageComponent {}
+export default class PageComponent {
+    protected readonly iconVariants = TUI_USED_ICONS;
```

Comment:
can you please add empty variant? like `''`

Reply:
✅